### ISSUE TITLE
Scaup: Providing profile data outside an ad response

### ISSUE THREAD

**Issue by brodrigu** [google/ads-privacy#26]
Scaup mentions ATPs being able to send user profile data in an ad response to be used for training the ML model. Will it be possible to also send user profile data outside of an ad response based on specific events such as the user visiting an advertiser site, or conversion?

**Comment by pjl-google** [comment]
Yes, that should be possible.  In the Scaup explainer we listed conversions as one event that’d be able to return data, in addition to ad requests - this is similar to the user-profile mechanism in SPURFOWL: https://github.com/AdRoll/privacy/blob/main/SPURFOWL.md

There’s no reason for conversions to be the only way to label the training data though - e.g. ad clicks could do the same thing.

**Comment by brodrigu** [comment]
That's good news. I recommend calling that out more explicitly in the proposal and then this issue could be closed.
